README.md

```diff
@@ -1,6 +1,193 @@
+
+In addition to neutralizing the "XML vs JSON" or more broadly "Markup language vs object notation" issue, this technology also offers other capabilities such as external specification and validation of application-oriented semantics ("business rules") defined over models appropriate to specialized knowledge domains.
+
+The current effort seeks to implement, demonstrate and enable the capabilities of a NIST Metaschema system, with respect to functionalities including schema generation, data conversion or pipelining, stylesheet generation or a host of other potential uses of Metaschema and metaschemas considered broadly. In addition, we aspire to do so sustainably, not in the sense that the software will meet all requirements for all time, but that it will be reliable, validable, traceable, extensible -- and replaceable.
+
+Considering data traceability and process transparency as primary goals will also serve secondary goals, such as the creation and demonstration of use cases (for this and compatible software) and the codification and maturation of relevant standards, including standards supported by Metaschema such as OSCAL.
+
+Stress is therefore on:
+
+- Transparency, testability and maintainability of processing on an open-source stack
+- Conformance in support of Metaschema-based data interchange / interop
+- Generality of support of Metaschema features
+
+What we do not emphasize:
+
+- Performance
+- Portability beyond the XML/XSLT dependency stack (instead relying on other Metaschema initiatives)
+- Use cases in isolation
+
+### Origins
+
+Formerly housed in the Metaschema repository, this code base traces the history of development of the Metaschema concept in the context of the OSCAL project. It was originally conceived as a demonstration and proof of concept, providing a bridge enabling JSON- and XML-based development in parallel over common problem sets and common data. Success in this effort led to a determination that multiple implementations of a platform-independent specification were needed, at which point this implementation was carved out into its own repository.
+
+### Project sunset
+
+Currently there is no plan to continue maintaining this project beyond the retirement of its lead researcher. A project goal is to enable the stabilization and socialization of the Metaschema technology, as evidenced by the use and support of *other* Metaschema implementations.
+
+The best way to ensure long-term access to the code base is to clone or fork the git repository and archive a copy for study or use.
+
+## Repository contents
+
+`src` includes XSLT source code, with supporting infrastructure including ad-hoc testing
+
+`support` includes dependent submodules with other static resources for configuration
+
+## Installation and use
```

Comment:
I suggest removing "and use" from the title since the next section is focusing on Using or Running it.

Reply:
"To run" is a subheading under "Installation and Use". But I'm trying for clarifying adjustments anyhow please check back after next commit. Will be much improved!